model-optimizer/extensions/ops/ONNXResize11.py

```diff
@@ -0,0 +1,72 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from mo.front.common.partial_infer.utils import int64_array
+from mo.graph.graph import Node, Graph
+from mo.ops.op import Op
+
+
+class ONNXResize11Op(Op):
+    op = 'ONNXResize11'
+
+    def __init__(self, graph: Graph, attrs: dict):
+        mandatory_props = {
+            'op': __class__.op,
+            'version': 'opset4',
```

Comment:
The `ONNXResize11` is an internal op so the `version` attribute should not be defined for it.

Reply:
Fixed.